ui/package.json

```diff
@@ -63,6 +63,7 @@
     "@icholy/duration": "^5.1.0",
     "@lineal-viz/lineal": "^0.5.1",
     "@tsconfig/ember": "^2.0.0",
+    "@types/codemirror": "^5.60.15",
```

Comment:
I thought I recall hearing that we would be getting rid of our usage of codemirror in favor of the HDS code editor component so perhaps we'll end up being able to remove this package but for now since the editor instance is passed as an arg on change we need it for ts components that use the `JsonEditor`.

Reply:
That's correct. There's potentially a chance we'll get to that in 1.20. But we'll see.